build.assets/Dockerfile

```diff
@@ -251,9 +251,11 @@ RUN make -C /opt/pam_teleport install
 ENV SOFTHSM2_PATH "/usr/lib/softhsm/libsofthsm2.so"
 
 # Install bats.
-RUN curl -fsSL https://github.com/bats-core/bats-core/archive/v1.2.1.tar.gz | tar -xz && \
-    cd bats-core-1.2.1 && ./install.sh /usr/local && cd .. && \
-    rm -r bats-core-1.2.1
+RUN git clone --depth=1 https://github.com/bats-core/bats-core.git -b v1.2.1 && \
```

Comment:
Why the change from downloading a tarball to git cloning the source?

Don't you think these changes belong in a separate PR?I don't see how they relate to removing the custom BoringSSL build.

Reply:
I was reviewing all the `Dockerfile`s to make sure I wasn't missing anything, and I noticed this discrepancy. I can split it out, but was kinda hoping to keep it as a /boyscout/ fix.